cg/models/demultiplex/flow_cell.py

```diff
@@ -126,6 +126,21 @@ def sequencer_type(
         """Return the sequencer type."""
         return sequencer_types[self.machine_name]
 
+    def get_bcl_converter(self, bcl_converter: str) -> str:
+        """
+        Return the BCL converter to use.
+        Tries to get the BCL converter from the sequencer type if not provided.
+        """
+        return bcl_converter or self.get_bcl_converter_by_sequencer()
+
+    def get_bcl_converter_by_sequencer(
+        self,
+    ) -> str:
+        """Return the BCL converter based on sequencer."""
+        if self.sequencer_type in [Sequencers.NOVASEQ, Sequencers.NOVASEQX]:
+            return BclConverter.DRAGEN
```

Comment:
We have two values for this conversion software: `BclConverter.DRAGEN` and `BclConverter.BCLCONVERT`. I think the new post processing flow relies on `BCLCONVERT` to be set. Might be worth double checking if this introduces some conflict.

I think we should remove one of the enum values so it is not duplicate because it can cause bugs and confusion. Might be out of scope for this PR though!

Reply:
When I looked at this some time ago, it seemed as if the software BCL convert did not care whether it is run on Dragen or not, i.e. it can be run on Dragen but it does not need to be that way (although maybe that is how it is set up at cg). So to me, BclConverter.DRAGEN represents something different than BCL Convert. Outside of the scope of this pr but I would say that the enums are not duplicated per se. 

Link for some documentation https://support.illumina.com.cn/bulletins/2020/10/upgrading-from-bcl2fastq-to-bcl-convert.html "The Illumina BCL Convert software is a standalone local Linux application"

Reply:
In the context of cg, BclConvert is always run in Dragen machines. There is no Bclconvert run in hasta. I I am not mistaken, the duplicate constant was indtroduced to slowly migrate from the constant `DRAGEN` to `BCLConvert`, which is more correct. I agree that it is time now to replace DRAGEN by BCLConvert, and that definitely has to be done in a different PR

Reply:
Yeah, it should not be called dragen. That is the name of the hardware platform. Not the bcl conversion software. So we should remove it from this enum and only use `BCLCONVERT`

Reply:
I guess the easiest would have been to just rename the dragen field to bclconvert

Reply:
I have an open issue regarding this. My aim was to do this once we have a more stable codebase around the demultiplexing and demux post processing. So yes, but not in PR.

Reply:
see #2030. So should we write that the issue is blocked by something? @ChrOertlin